Wikipedia/Code/WebViewController.m

```diff
@@ -133,6 +133,9 @@ - (void)viewDidLoad {
     [self displayArticle];
 }
 
+NSString* const WMF_UIScrollViewDidEndDeceleratingNotification = @"_UIScrollViewDidEndDeceleratingNotification";
```

Comment:
@coreyfloyd viewwilldissapear doesn't do what we need, for instance - it gets called when you tap a link which causes a new view to be pushed over the old one, and we don't want to record the position of the old article in this case because the last read article is now the new one which was just pushed. Additionally, viewwilldissapear isn't called when the app resigns or is killed afaict.